lib/edit-site-page.php

```diff
@@ -137,11 +137,6 @@ function gutenberg_edit_site_init( $hook ) {
 	$settings['styles'] = gutenberg_get_editor_styles();
 	$settings           = gutenberg_experimental_global_styles_settings( $settings );
 
-	// This is so other parts of the code can hook their own settings.
```

Comment:
Note how it the line above this code we call `gutenberg_experimental_global_styles_settings`.